charts/postgres-db-sync/templates/sync-job.yaml

```diff
@@ -77,34 +82,23 @@ spec:
                   fieldRef:
                     fieldPath: status.podIP
               - name: PROMETHEUS_PUSHGATEWAY_URL
-                value: {{ .Values.PrometheusPushGatewayURL }}
+                value: {{ .Values.prometheusPushGatewayURL }}
               - name: CRONJOB_NAME
                 value: {{ include "pgdbsync.fullname" . }}
               {{- end }}
             volumeMounts:
               - name: config-volume
                 mountPath: /usr/local/bin
-              - name: export-query
-                mountPath: /usr/src/pg-db-sync/export.sql
-                subPath: export.sql
-              {{- if .Values.createTargetTableConfigMap }}
-              - name: create-target-table
-                mountPath: /usr/src/pg-db-sync/create_target.sql
-                subPath: create_target.sql
-              {{- end }}
+              - name: export-queries
+                mountPath: /usr/src/pg-db-sync
           volumes:
```

Comment:
I don't think we can do that (deleting the if condition): it is a regression not to be able to provide the query for the table creation.
Not being able to select that, you may face issues with the timestamp implementation: you will always have the `create_target.sql` file present, so it will always be ran in the timestamp implementation (as it is only checking if the file is present to run the query). 
We also do not know if we already have some use cases which are using this feature. So it is a breaking change on a repo without tags/versions/etc...